function/pkg/operator/subscription_operator.go

```diff
@@ -2,27 +2,22 @@ package operator
 
 import (
 	"context"
+	"errors"
 
 	"github.com/kyma-incubator/hydroform/function/pkg/client"
 	"github.com/kyma-incubator/hydroform/function/pkg/resources/types"
-	"github.com/pkg/errors"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-type functionReference struct {
-	name      string
-	namespace string
-}
-
-type triggersOperator struct {
+type subscriptionOperator struct {
 	fnRef functionReference
 	items []unstructured.Unstructured
 	client.Client
 }
 
-func NewTriggersOperator(c client.Client, fnName, fnNamespace string, u ...unstructured.Unstructured) Operator {
-	return &triggersOperator{
+func NewSubscriptionsOperator(c client.Client, fnName, fnNamespace string, u ...unstructured.Unstructured) Operator {
```

Comment:
Sometimes you use 'subscriptions' , sometimes you use 'subscription'. 
I think we should use one form.